packages/components/nodes/chatmodels/AzureChatOpenAI/AzureChatOpenAI.ts

```diff
@@ -0,0 +1,122 @@
+import { OpenAIBaseInput } from 'langchain/dist/types/openai-types'
+import { INode, INodeData, INodeParams } from '../../../src/Interface'
+import { getBaseClasses } from '../../../src/utils'
+import { AzureOpenAIInput, ChatOpenAI } from 'langchain/chat_models/openai'
+
+class AzureChatOpenAI_ChatModels implements INode {
+    label: string
+    name: string
+    type: string
+    icon: string
+    category: string
+    description: string
+    baseClasses: string[]
+    inputs: INodeParams[]
+
+    constructor() {
+        this.label = 'Azure ChatOpenAI'
+        this.name = 'azureChatOpenAI'
+        this.type = 'AzureChatOpenAI'
+        this.icon = 'Azure.svg'
+        this.category = 'Chat Models'
+        this.description = 'Wrapper around Azure OpenAI large language models that use the Chat endpoint'
+        this.baseClasses = [this.type, ...getBaseClasses(ChatOpenAI)]
+        this.inputs = [
+            {
+                label: 'Azure OpenAI Api Key',
+                name: 'azureOpenAIApiKey',
+                type: 'password'
+            },
+            {
+                label: 'Model Name',
+                name: 'modelName',
+                type: 'options',
+                options: [
+                    {
+                        label: 'gpt-4',
+                        name: 'gpt-4'
+                    },
+                    {
+                        label: 'gpt-4-32k',
+                        name: 'gpt-4-32k'
+                    },
+                    {
+                        label: 'gpt-35-turbo',
+                        name: 'gpt-35-turbo'
+                    }
+                ],
+                default: 'gpt-35-turbo',
+                optional: true
+            },
+            {
+                label: 'Temperature',
+                name: 'temperature',
+                type: 'number',
+                default: 0.9,
+                optional: true
+            },
+            {
+                label: 'Azure OpenAI Api Instance Name',
```

Comment:
Same, make sure just include fields that are needed: https://js.langchain.com/docs/modules/models/chat/integrations